bigquery_etl/cli/check.py

```diff
@@ -0,0 +1,199 @@
+"""bigquery-etl CLI check command."""
+import datetime
+import re
+import subprocess
+import sys
+import tempfile
+from subprocess import CalledProcessError
+
+import click
+
+from ..cli.utils import is_authenticated, paths_matching_checks_pattern, sql_dir_option
+from ..util.common import render as render_template
+
+
+def _build_query_arguments(parameters):
+    """
+    Convert a dict of parameters (from CLI) into query arguments.
+
+    Opposite of _build_query_arguments.
+    """
+    if parameters is None:
+        return []
+    return [f"--parameter={key}::{value}" for key, value in parameters.items()]
+
+
+def _parse_query_arguments(query_args):
+    parameters = []
+    for query_arg in query_args:
+        if query_arg.startswith("--parameter=") and len(query_arg.split("=")) == 2:
+            parameters.append(query_arg.split("=")[1])
+        else:
+            raise ValueError("argument must start with --parameter")
```

Comment:
does this mean `--use_legacy_sql=false` as query argument is not supported?

Reply:
latest updates fully support all formats.

Reply:
If we use query parameters this flag needs to be passed `--use_legacy_sql=false`, I wonder could we just implicitly always pass that flag without me as the user having to include it in the command?

Reply:
`--use_legacy_sql=false` is always added in `_run_check()`